src/components/organisms/AssetActions/Pool/Remove.module.css

```diff
@@ -60,3 +60,34 @@
 .output figure[class*='pool shares'] {
   display: none;
 }
+
+.slippage {
```

Comment:
these are the exact same styles as https://github.com/oceanprotocol/market/blob/main/src/components/organisms/AssetActions/Trade/Slippage.module.css

If we do not want to use the existing `Slippage` component and make it usable for both use cases (which would be preferred), we should at least not repeat that many styles and rather make use of CSS modules and `compose` them from `Trade/Slippage.module.css` into here, e.g.:

```css
.slippage {
  composes: slippage from '../Trade/Slippage.module.css'
}
```